modules/client-configuration/pages/supported-features-centos.adoc

```diff
@@ -15,59 +15,215 @@ The operating system you run on a client is supported by the organization that s
-| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"] | ``Salt``
-| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt`` | ``Salt``
-| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"] | ``Salt``
-| System deployment (PXE/Kickstart) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| System redeployment (Kickstart) | ``Traditional`` | icon:check[role="green"] | ``Salt``
-| Contact methods | ``Traditional``: OSAD, RHNSD, SSH-push. +
-``Salt``: ZeroMQ, Salt-SSH | ``Traditional``: OSAD, RHNSD, SSH-push. ``Salt``: ZeroMQ, Salt-SSH | ``Salt``: ZeroMQ, Salt-SSH
-| Works with {productname} Proxy | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Action chains | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Software crash reporting | icon:times[role="danger"] | ``Traditional`` | icon:times[role="danger"]
-| Staging (pre-download of packages) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Duplicate package reporting | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| CVE auditing (requires CVE ID) |  icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| SCAP auditing | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Package verification | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| Package locking | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| System locking |  ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| System snapshot | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
-| Configuration file management |  icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Snapshots and profiles |  ``Traditional``. ``Salt``: Profiles supported, Sync not supported | ``Traditional``. ``Salt``: Profiles supported, Sync not supported | ``Salt``: Profiles supported, Sync not supported
-| Power management |  icon:check[role="green"] | icon:check[role="green"] | ``Salt``
-| Monitoring |  ``Salt`` | ``Salt`` | ``Salt``
-| Docker buildhost |  icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
-| Build Docker image with OS | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
-| Kiwi buildhost |  icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
-| Build Kiwi image with OS |  icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
+
+| Feature
+| {centos}{nbsp}6
+| {centos}{nbsp}7
+| {centos}{nbsp}8
+
+| Client
+| {question} (plain CentOS) / {check} (with {es})
+| {question} (plain CentOS) / {check} (with {es})
+| {question} (plain CentOS) / {salt} (with {es})
```

Comment:
With 4.1 plain CentOS is supported by us. So I think the {question} can go away, right?
We also have a special "Expanded Support" section. Is there really "(with {es})" needed? Same as for Native Red Hat